src/edu/csus/ecs/pc2/ui/team/CreateTeamsTSV.java

```diff
@@ -0,0 +1,268 @@
+// Copyright (C) 1989-2021 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.ui.team;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.ParseArguments;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.exception.CommandLineErrorException;
+import edu.csus.ecs.pc2.core.imports.clics.TeamAccount;
+
+/**
+ * Read teams.json write teams.tsv
+ * 
+ * @author pc2@ecs.csus.edu
+ * @version $Id$
+ */
+
+public class CreateTeamsTSV {
+
+    public static final String GROUPS_TSV_FILENAME = "groups.tsv";
+
+    private static final String TAB = "\t";
+
+    public static final String TEAMS_TSV_FILENAME = "teams.tsv";
+
+    public static final String TEAMS_JSON_FILENAME = "teams.json";
+
+    private static final String OUTPUT_FILE_KEY = "--of";
+    
+    private static final String INPUT_FILE_KEY = "--if";
+    
+    private static final String OVERWRITE_FILE_KEY = "-f";
+
+    /**
+     * Successful run exit code.
+     *
+     * Using a non-zero exit code because if there is a problem in the JVM or elsewhere a zero exit code could be returned.
```

Comment:
I really don't understand the logic behind this statement.  "a problem in the JVM or elsewhere" could also return an exit code of 5, so it seems like the argument being made is fallacious -- in which case it seems like it would be better to at least use the standard convention of 0=success.

Reply:
A JVM can return 0 when there is an error/Exception.
To avoid a false success a alternate success code is used.